### PR TITLE
Add analysis comment column

### PR DESCRIPTION
**Pull request by ivadym**
## Description

Part of https://github.com/Clinical-Genomics/cg/issues/2702

### Added

- Force storage comment column to analysis table
- Force storage comment to delivery report

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### Expected test outcome

- [x] Comment column in analysis table
Screenshot 2024-07-29 at 11 31 58

- [x] Delivery report including comment as a warning
Screenshot 2024-07-29 at 12 19 20

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by ChrOertlin — APPROVED** [review]
👍

File: alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""Add comment column to analysis table
+
+Revision ID: 80bac9d48872
+Revises: 0808675ad136
```
Comment (ChrOertlin) [inline alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py:4]:
```suggestion
Revises: 77a75121be31
```

File: alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "80bac9d48872"
+down_revision = "0808675ad136"
```
Comment (ChrOertlin) [inline alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py:15]:
```suggestion
down_revision = "77a75121be31"
```

File: alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py
Comment (ChrOertlin) [inline alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py:1]: if you can change the date to when this is merged, that would be nice!

Comment (ivadym) [reply alembic/versions/2024_07_26_80bac9d48872_add_comment_column_to_analysis_table.py:1]: I will, thanks for letting me know!

**Pull request merged**